tests/cors.js

```diff
@@ -0,0 +1,36 @@
+const { it } = require('mocha');
+const { expect } = require('chai');
+const request = require('./base');
+const { corsOptions } = require('../utils/cors');
+
+const expectCORSHeaders = (response) => {
```

Comment:
```suggestion
const expectCORSHeaders = response => {
```
Sorry to be pedantic, but I like to have consistency :)

Reply:
Seems like something `--fix` should fix.